app/forms/Ducr.scala

```diff
@@ -26,7 +26,11 @@ case class Ducr(ducr: String)
 object Ducr {
   implicit val format = Json.format[Ducr]
 
-  private val ducrFormat = "^\\d[A-Z]{2}\\d{12}-[0-9A-Z]{1,19}$"
+  private val ducrFormat = """^[0-9][A-Z][A-Z][0-9A-Z\(\)\-/]{6,32}|
```

Comment:
Where is specification for this regexp ?

Reply:
Is in a tariff document.
They accept 5 different DUCR formats.
I am just getting this confirmed with Ian.

We have used the same for the movements project.

Reply:
Would you like to put this as scaladoc over - just to not lose connection from where it comes.

Reply:
This has been addressed.